cmd/installer/cli/install.go

```diff
@@ -780,6 +779,24 @@ func validateAdminConsolePassword(password, passwordCheck string) bool {
 	return true
 }
 
+func replicatedAppURL() string {
+	var embCfgSpec *ecv1beta1.ConfigSpec
+	if embCfg := release.GetEmbeddedClusterConfig(); embCfg != nil {
+		embCfgSpec = &embCfg.Spec
+	}
+	domains := runtimeconfig.GetDomains(embCfgSpec)
+	return netutil.MaybeAddHTTPS(domains.ReplicatedAppDomain)
+}
+
+func proxyRegistryURL() string {
+	var embCfgSpec *ecv1beta1.ConfigSpec
+	if embCfg := release.GetEmbeddedClusterConfig(); embCfg != nil {
+		embCfgSpec = &embCfg.Spec
+	}
+	domains := runtimeconfig.GetDomains(embCfgSpec)
+	return netutil.MaybeAddHTTPS(domains.ProxyRegistryDomain)
+}
```

Comment:
why aren't these in the runtimeconfig package?

For that matter, if you're not going to return an error here, why not do that for a GetDomains function?

Reply:
it's an intentional decision to put these here in the cli package because they use the embedded config spec, whereas other packages might have to use the installation config spec, and i didn't want us getting this from runtimeconfig by mistake.

Reply:
because that's hard to read :)

Reply:
also, `GetDomains` takes a config spec, not a config